ci-queue.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency 'bundler', '~> 1.13'
   spec.add_development_dependency 'rake', "~> 10.0"
-  spec.add_development_dependency 'minitest', '~> 5.0'
+  spec.add_development_dependency 'minitest', '~> 5.9.1'
```

Comment:
Minitest released a `5.10.0` that breaks minitest-reporters, the fix is on the way upstream: https://github.com/kern/minitest-reporters/pull/220